jsonnet/rules.libsonnet

```
@@ -581,5 +581,18 @@ function(params) {
         },
       ],
     },
+    {
+      name: 'openshift-logging.rules',
+      rules: [
+        {
+          expr: 'sum(log_logged_bytes_total{container="",pod!=""}[5m]) BY (cluster)',
+          record: 'cluster:log_logged_bytes_total:sum',
+        },
+        {
+          expr: 'sum(log_collected_bytes_total{container="",pod!=""}[5m]) BY (cluster)',
+          record: 'cluster:log_collected_bytes_total:sum',
```

The rule expression is not valid PromQL.
```suggestion
          expr: 'sum(log_logged_bytes_total{container="",pod!=""}) BY (cluster)',
          record: 'cluster:log_logged_bytes_total:sum',
        },
        {
          expr: 'sum(log_collected_bytes_total{container="",pod!=""}) BY (cluster)',
          record: 'cluster:log_collected_bytes_total:sum',
```
would be valid PromQL. Please double check this rule actually does what you want.

---

resolved